.github/workflows/build.yaml

```diff
@@ -330,7 +331,7 @@ jobs:
     runs-on: ${{matrix.os}}
     steps:
       - name: Install dependencies
-        run: sudo apt update && sudo apt install -y liblzo2-dev libpam0g-dev liblz4-dev libcap-ng-dev libnl-genl-3-dev linux-libc-dev man2html libcmocka-dev python3-docutils libtool automake autoconf ${SSLPKG}
+        run: sudo apt update && sudo apt install -y liblzo2-dev libpam0g-dev liblz4-dev libcap-ng-dev libnl-genl-3-dev linux-libc-dev man2html libcmocka-dev python3-docutils libtool automake autoconf ${SSLPKG} libpkcs11-helper1-dev gnutls-bin softhsm2
```

Comment:
Anyway, currently the test works only with OpenSSL 3.0 because of the way sign/verify operation are done. May not be hard to extend it to OpenSSL 1.1, but I would ignore 1.0.2.